<ui version="4.0" >
 <class>AboutDlg</class>
 <widget class="QDialog" name="AboutDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>About FatRat</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QTableWidget" name="tableMenu" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize" >
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode" >
      <enum>Qt::ElideMiddle</enum>
     </property>
     <property name="showGrid" >
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled" >
      <bool>false</bool>
     </property>
     <property name="rowCount" >
      <number>4</number>
     </property>
     <property name="columnCount" >
      <number>1</number>
     </property>
     <row/>
     <row/>
     <row/>
     <row/>
     <column/>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QStackedWidget" name="stackedWidget" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="page" >
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label" >
         <property name="pixmap" >
          <pixmap resource="gfx/resources.qrc" >:/fatrat/fatrat_brown.png</pixmap>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>&lt;font size=20>FatRat download manager</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="labelVersion" >
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Copyright © 2006-2008 Luboš Doležel&lt;br>&lt;a href="http://fatrat.dolezel.info">http://fatrat.dolezel.info&lt;/a></string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2" >
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item row="0" column="0" >
        <widget class="QTextBrowser" name="textLicense" >
         <property name="font" >
          <font>
           <family>Courier New</family>
          </font>
         </property>
         <property name="lineWrapMode" >
          <enum>QTextEdit::NoWrap</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3" >
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item row="0" column="0" >
        <widget class="QTextBrowser" name="textTranslators" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_4" >
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item row="0" column="0" >
        <widget class="QTextBrowser" name="text3rdParties" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="gfx/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AboutDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>318</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AboutDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>386</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableMenu</sender>
   <signal>currentCellChanged(int,int,int,int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>56</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel" >
     <x>183</x>
     <y>10</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
